docs/manuals/running_services/index.md

```diff
@@ -0,0 +1,90 @@
+# Running Taipy services
+
+Taipy provides you three services: Taipy Gui, Taipy Rest, and Taipy Core. You can run Taipy services by calling the method _run()_ from the service instance either `Gui^`, `Rest^`, or `Core^`. You can also use `taipy.run()^` to initiate the service.
```

Comment:
Gui -> GUI

"from the service instance either `Gui^`, `Rest^`, or `Core^`"
->
"from of a service instance that can be either `Gui^`, `Rest^`, or `Core^`"

"to initiate the service"
->
"to initiate services"